src/console_reporter.cc

```diff
@@ -142,10 +142,16 @@ void ConsoleReporter::PrintRunData(const Run& result) {
   } else if (result.report_rms) {
     printer(Out, COLOR_YELLOW, "%10.0f %-4s %10.0f %-4s ", real_time * 100, "%",
             cpu_time * 100, "%");
-  } else {
+  } else if (result.run_type != Run::RT_Aggregate ||
+             result.aggregate_unit == StatisticUnit::Time) {
     const char* timeLabel = GetTimeUnitString(result.time_unit);
     printer(Out, COLOR_YELLOW, "%s %-4s %s %-4s ", real_time_str.c_str(), timeLabel,
             cpu_time_str.c_str(), timeLabel);
+  } else {
+    assert(result.aggregate_unit == StatisticUnit::Percentage);
+    printer(Out, COLOR_YELLOW, "%10.2f %-4s %10.2f %-4s ",
+            (100. * result.real_accumulated_time), "%",
```

Comment:
bit awkward this is called `_time` and now isn't always one. maybe a rename? or is that too intrusive?

Reply:
I keep delaying dealing with this, so i still don't want to change this now,
but this should be a vector of measurements, just two hardcoded "timers" isn't right.